sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/e-cert-utils.ts

```diff
@@ -139,6 +139,7 @@ export async function createBlockedDisbursementTestData(
     isValidSIN?: boolean;
     disbursementValues?: DisbursementValue[];
     msfaaState?: MSFAAStates;
+    firstDisbursementInitialValues?: Partial<DisbursementSchedule>;
```

Comment:
I would say instead of receiving a new parameter, set `hasEstimatedAwards` based on the size of the disbursement values to keep it simple. 

![image](https://github.com/user-attachments/assets/47cf3e39-8479-473d-a781-5b67917b150f)

Reply:
Or even as a better solution, I would suggest to adjust the factory `createFakeDisbursementSchedule` to set the `hasEstimatedAwards` based on disbursement values. Otherwise you have to go and set the flag as false for existing tests and any tests that will be written in future. 

![image](https://github.com/user-attachments/assets/864e1a32-6cf5-41fc-86ad-714b7c4e8e72)